lambda/lambda.ml

```diff
@@ -736,9 +741,9 @@ let subst update_env ?(freshen_bound_variables = false) s input_lam =
       ) ids ([], l)
   in
   let bind_rec ids l =
-    List.fold_right (fun (id, clas, def) (ids', l) ->
+    List.fold_right (fun { id; rkind; def } (ids', l) ->
         let id', l = bind id l in
-        ((id', clas, def) :: ids' , l)
+        ({ id = id'; rkind; def } :: ids' , l)
```

Comment:
I would prefer a `{ binding with id = id' }` approach, which requires no changes when adding new fields in the future.